.pre-commit-config.yaml

```diff
@@ -40,3 +40,15 @@ repos:
     hooks:
       - id: black
         language_version: python3.9
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.3.0
+    hooks:
+      - id: remove-tabs
+        types: [file]
+        files: \.talon$
+        args: ["--whitespaces-count=4"]
```

Comment:
Huh, it does seem like most of our .talon files use 4 space indents. We should fix the .editorconfig to do the same.

Reply:
Opened #970 for this.

Reply:
Would be cool if remove-tabs respected .editorconfig.  I believe prettier does